build.sbt

```diff
@@ -10,8 +10,11 @@ scalaVersion := "2.10.4"
 
 libraryDependencies += "org.apache.hadoop" % "hadoop-client" % "1.0.4"
 
-libraryDependencies += "org.apache.spark" %% "spark-sql" % "1.2.0"
+libraryDependencies += "org.apache.spark" %% "spark-sql" % "1.3.0-SNAPSHOT"
```

Comment:
Actually, that might not be a great idea since the snapshot artifacts have the version "1.3.0" even though they might not reflect the final 1.3.0 that gets released, so this could cause RCs to get cached in user Maven / Ivy caches and stay there even after 1.3.0 final is released.